examples/models/llama2/export_llama_lib.py

```diff
@@ -495,9 +502,18 @@ def _export_llama(modelname, args) -> LLMEdgeManager:  # noqa: C901
         )
         # pyre-ignore: Undefined import [21]: Could not find a module corresponding to import `executorch.backends.qualcomm.utils.utils`
         from executorch.backends.qualcomm.utils.utils import _transform
+        from executorch.exir.program._program import _get_updated_graph_signature
 
         # pyre-ignore: Undefined attribute [16]: Module `executorch.backends` has no attribute `qualcomm`, Optional type has no attribute `exported_program`
         _transform(builder_exported_to_edge.edge_manager.exported_program())
+        # Since QDQ nodes are stripped in QNN, update graph signature again to validate program
+        builder_exported_to_edge.edge_manager.exported_program()._graph_signature = (
```

Comment:
Is it possible to have them as part of the `_transform()`

Reply:
Good idea. Thanks